### ISSUE TITLE
Move "occurrences" and "refresh" attribute reference documentation 

### ISSUE THREAD

**Issue by calebhailey** [sensu/sensu-docs#640]
Documentation for "occurrences" and "refresh" is currently found on the Plugins reference documentation page, rather than the Checks documentation page. The reasoning behind this was because "occurrences" and "refresh" were historically not formally supported by Sensu itself, but rather, by the `sensu-plugin` gem. This [changed in version 0.26](https://sensuapp.org/docs/0.26/overview/changelog.html#core-v0-26-0) when we began shipping the built-in "occurrences" filter. 

We even explain the original reasoning [in the Plugin reference documentation](https://sensuapp.org/docs/latest/reference/plugins.html#sensu-plugin-definition-specification) as: 

> _NOTE: plugins based on the sensu-plugin gem derive configuration from [custom check definition attributes](https://sensuapp.org/docs/latest/reference/checks.html#custom-attributes). The configuration example(s) provided above, and the “specification” provided here are for clarification and convenience only (i.e. this “specification” is just an extension of the [check definition specification](https://sensuapp.org/docs/latest/reference/checks.html#check-definition-attributes), and not a definition of a distinct Sensu primitive)._

Basically, this note is no longer true, so we can remove it and move the reference docs to the check definition specification section.

**Comment by majormoses** [comment]
Hm, agreed it needs to be updated the question is how to best convey the current state. I would say it is still not part of the core sensu and is delivered by an extension that is bundled with sensu. This also requires you to declare the filter to each handler so I do feel like in my mind it is still a custom attribute rather than is something natively in sensu that requires no additional changes to config to get it to work. Feel free to disagree or say I am bikeshedding on verbiage. Just my 2 cents.

**Comment by majormoses** [comment]
To further clarify, what I mean to say is that it is not a native feature of sensu's framework but is a custom extension (even if it is bundled with core by default) therefore it is still a custom attribute. To me the distinction on a custom attribute is about whether it is part of the framework vs an extension (not as in running inside sensu runtime but filters, handlers, mutators, check scripts, metric scripts, etc) of it,

**Comment by majormoses** [comment]
I don't think it should be removed as it still rings true other than it referring to `sensu-plugin` rather than `sensu-extensions-occurrences`.

**Comment by cwjohnston** [comment]
This issue was moved to sensu/sensu-docs-site#120
